csuntechlab/iSTART
# STR-148 -- FE: Implement Multi-Question text-input (1-100) Quiz Slide + Answer Slide #8, #9

# Description
Implemented input comparison and input results template.
  
# Testing
1. Login as `cyndee.bellamy` or `april.feldman`.
2. Click the alcohol module in dashboard.
3. Visit slide #8 and #9 and make sure all the features from the document and mockup conform with the content provided.
